app-cdr/cdrtools/cdrtools-3.01~a30.recipe

```diff
@@ -0,0 +1,166 @@
+SUMMARY="cdrtools is a CD/DVD/Blu-ray premastering and recording software"
```

Comment:
@waddlesplash: Just fixed.
BTW, should I discard https://github.com/fbrosson/haikuports/commit/a3ee59c1a3b98777d1a4bbd57f7b5c12ef1e4ec8#diff-144005c153d6a7daab3dbabf39a70059 before merging my commits or may merge them all ?